tests/test_database_to_staging.py

```diff
@@ -886,3 +885,143 @@ def test_store_mutation_in_cis_files_to_staging_no_centers(
+    database_to_staging.SV_CENTER_PATH = os.path.join(
+        database_to_staging.GENIE_RELEASE_DIR, "data_sv_%s.txt"
+    )
+    with patch("builtins.open") as mock_open, patch.object(
+        extract, "get_syntabledf", return_value=svdf
+    ) as patch_get_syntabledf, patch.object(
+        syn, "create_snapshot_version", return_value=1
+    ), patch.object(
+        pd.DataFrame, "to_csv"
+    ) as patch_to_csv, patch.object(
+        load, "store_file"
+    ) as patch_store_file, patch.object(
+        process_functions, "removePandasDfFloat", return_value="test_sv_text"
+    ):
+        # call the function
+        sv_sample = database_to_staging.store_sv_files(
+            syn,
+            release_synid="syn123",
+            genie_version="TESTING",
+            synid="syn5678",
+            keep_for_center_consortium_samples=["GENIE-1", "GENIE-2"],
+            keep_for_merged_consortium_samples=["GENIE-1", "GENIE-3"],
+            current_release_staging=current_release_staging,
+            center_mappingdf=center_mappingdf,
+        )
+
+        # validate the calls
+        patch_get_syntabledf.assert_called_once_with(syn, "select * from syn5678")
+        if not current_release_staging:
+            patch_to_csv.assert_has_calls(
+                [
+                    mock.call(
+                        database_to_staging.SV_CENTER_PATH % "C1", sep="\t", index=False
+                    ),
+                    mock.call(
+                        database_to_staging.SV_CENTER_PATH % "C2", sep="\t", index=False
+                    ),
+                ]
+            )
+
+            # Check the store_file call arguments
+            patch_store_file.assert_has_calls(
+                [
+                    mock.call(
+                        syn=syn,
+                        filepath=database_to_staging.SV_CENTER_PATH % "C1",
+                        version_comment="TESTING",
+                        parentid="syn123",
+                    ),
+                    mock.call(
+                        syn=syn,
+                        filepath=database_to_staging.SV_CENTER_PATH % "C2",
+                        version_comment="TESTING",
+                        parentid="syn456",
+                    ),
+                ]
+            )
+
+        mock_open.assert_called_once_with(
+            os.path.join(database_to_staging.GENIE_RELEASE_DIR, "data_sv.txt"), "w"
+        )
+        file_handle = mock_open.return_value.__enter__.return_value
+        file_handle.write.assert_called_with("test_sv_text")
+        patch_store_file.assert_called_with(
+            syn=syn,
+            filepath=os.path.join(database_to_staging.GENIE_RELEASE_DIR, "data_sv.txt"),
+            parentid="syn123",
+            version_comment="TESTING",
+            name="data_sv.txt",
+            used="syn5678.1",
+        )
+        assert sv_sample == ["GENIE-1", "GENIE-3"]
+
+
+def test_store_data_gene_matrix(syn):
```

Comment:
Maybe other test case(s) where nothing should get filtered out in the data gene matrix in the `sv` columns / no wes panels ids are present in current data for example